### DISCUSSION TITLE
Fill a region with smoke as an initial condition

### DISCUSSION THREAD

**Discussion by SaiSantosh1824** [firemodels/fds#11751] [Announcements]
Dear All,

I am relatively new to Pyrosim. Kindly help me figure out how to fill a region with smoke as an initial conditions and then use induction fans to clear the region. Upon searching the internet, I came across an expired link (https://www2.thunderheadeng.com/2015/04/modeling-jet-fans-fds-part-3-applications/) which provides a tutorial file to do the same. 

It would be really helpful if some one has a copy of the file or if someone can elaborate this methodology.

Thank you!!!

**Comment by drjfloyd** [comment]
Pyrosim is a commercial tool that uses FDS (the software in this github repo). Thunderhead Engineering provides technical support for specific questions on using Pyrosim.

For learning how FDS works, the first step is to read the FDS documentation. A detailed User's Guide, Technical Reference Guide, Verification Guide, Validation Guide, and Configuration Management Plan is provided when FDS is installed (inclding when FDS is installed as part of the Pyrosim installation process).

**Reply by SaiSantosh1824** [reply]
Thank you for the response!
Can you please provide some guidance on smoke logging a region as an initial condition to then clear it with an induction fan.
I am aware to do it by starting a fire and stopping the fire after some time to fill the region. But the initial condition will help to save some time.
Kindly provide guidance for this, specific to FDS as command lines!

**Comment by drjfloyd** [comment]
![image](https://user-images.githubusercontent.com/12799217/232524755-52aad1ef-0d51-4f45-865a-6661bbb8b6b1.png)

**Reply by SaiSantosh1824** [reply]
Thank you for the update!

**Comment by bwklein** [comment]
This should not be in the Announcements category.
